Cargo.toml

```diff
@@ -11,6 +11,8 @@ members = [
   "examples/sprites",
   "examples/todolist",
   "examples/wrapper-types",
+  "examples/json-wrapper-lib",
+  "examples/json-wrapper-ext",
```

Comment:
That's the idea or maybe multiple crates.  One motivation for this is `Timestamp` on python.  When I was working on that I thought it would be nice to offer a localized, timezone-aware, version.  But that requires the `pytz` library and it didn't seem right to require that for a core type.  But it would be great to have an extension crate that offered that.